### DISCUSSION TITLE
Features list

### DISCUSSION THREAD

**Discussion by not-nef** [sumeshir26/TimerX#82] [Ideas]
I made a features list, maybe we could integrate it into the readme? ![image](https://user-images.githubusercontent.com/83908932/152693256-76f057cf-4ae9-4891-b9e8-92b08348f75b.png)

**Comment by not-nef** [comment]
I also added a planned features list ![image](https://user-images.githubusercontent.com/83908932/152693485-b33fcf5c-2f74-4cab-82a9-e54530871f24.png)

**Comment by lg-nineoneninethreenine** [comment]
@sumeshir26 your idea?

**Reply by sumeshir26** [reply]
Waiting for your PR

**Reply by lg-nineoneninethreenine** [reply]
I'll do this by 12 pm IST, got an exam goin' on.

**Reply by not-nef** [reply]
> I'll do this by 12 pm IST, got an exam goin' on.

Good luck

**Reply by lg-nineoneninethreenine** [reply]
Thanks, almost done with this.

**Comment by lg-nineoneninethreenine** [comment]
Also, can we release a patch release by this month end so that we could work on the other features after March 13-16? I bet we'll have exams during that time. Also, let's look up for a good April Fools easter egg.

**Comment by sumeshir26** [comment]
My exams start fdrom April 3 so i reallay wnt to get out v1.1 before that

**Comment by lg-nineoneninethreenine** [comment]
:+1: so why not Feb28?

**Comment by sumeshir26** [comment]
I mean march 3 not april
Also, what app are you using? I want to get it on my iPad

**Reply by lg-nineoneninethreenine** [reply]
Oh that, I think it's Obsidian. Famous for Markdown.

**Comment by not-nef** [comment]
> Also, can we release a patch release by this month end so that we could work on the other features after March 13-16? I bet we'll have exams during that time. 

Yep. Math 😩

**Reply by lg-nineoneninethreenine** [reply]
Math is fun for me. It's just general science that irritates me.

**Reply by not-nef** [reply]
I generally have no problem with it theres just some things i dont like

**Reply by not-nef** [reply]
I also have a latin exam soon and thats gonna suck

**Reply by not-nef** [reply]
Anyway im in school now

**Reply by lg-nineoneninethreenine** [reply]
All the best for that!

**Reply by sumeshir26** [reply]
For me, math is OK, science is the hard one. My other subjects(English, German, Hindi, Social Science, IT) are fine. Also, before we do a release, I want to make a Upgrade Path for TimerX
